android-sdk/src/androidTest/java/com/bitmark/sdk/test/integrationtest/DataProvider.java

```diff
@@ -8,7 +8,7 @@
  * @author Hieu Pham
  * @since 3/21/19
  * Email: hieupham@bitmark.com
- * Copyright © 2019 Bitmark. All rights reserved.
+ * Copyright © 2014-2019 Bitmark. All rights reserved.
```

Comment:
@author should go after the license!!!!

Where is the SPDX tag before the license?
Also, where is the license after the copyright?